js/full-picture.js

```diff
@@ -0,0 +1,56 @@
+import { isEscapeKey } from './utils.js';
+import { DESCRIPTIONS } from './data.js';
+import { getRandomArrayElement } from './utils.js';
+import { createComments } from './comments.js';
+import { commentsArray } from './data.js';
+
+export const interactWithBigPicture = () => {
+  const pictureElement = document.querySelector('.pictures');
+  const bigPictureShow = document.querySelector('.big-picture');
+  const bigPictureClose = document.querySelector('.big-picture__cancel');
+  const bigPictureUrl = bigPictureShow.querySelector('.big-picture__img').children;
```

Comment:
```suggestion
  const bigPictureImage = bigPictureShow.querySelector('.big-picture__img img');
```
`children` возвращает коллекцию элементов, а мы ожидаем увидеть один элемент